chipsec/cfg/8086/mtl.xml

```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0"?>
+<configuration platform="MTL" req_pch="True">
+<!--
+CHIPSEC: Platform Security Assessment Framework
+Copyright (c) 2024, Intel Corporation
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; Version 2.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+Contact information:
+chipsec@intel.com
+-->
+
+<!--
+XML configuration file for MeteorLake based platforms
+
+http://www.intel.com/content/www/us/en/processors/core/core-technical-resources.html
+https://cdrdv2.intel.com/v1/dl/getContent/795249
+
+* Intel(R) Core Ultra Processor Family Datasheet
+-->
+
+<!-- MTL EDS Volume 1: 640228 2.3 Table 7 -->

```

Comment:
EDS # should be removed and replaced with Datasheet # 792044 (section and table # match)